optaplanner-test/src/main/java/org/optaplanner/test/api/score/stream/ConstraintVerifier.java

```diff
@@ -50,6 +53,30 @@
         return new DefaultConstraintVerifier<>(constraintProvider, solutionDescriptor);
     }
 
+    /**
+     * Alternative to {@link #build(ConstraintProvider, Class, Class[])}.
+     *
+     * @param solverConfig never null, must have a {@link PlanningSolution} class, {@link PlanningEntity} classes
+     *        and a {@link ConstraintProvider} configured.
+     * @param <ConstraintProvider_> type of the {@link ConstraintProvider}
+     * @param <Solution_> type of the {@link PlanningSolution}-annotated class
+     * @return never null
+     */
+    static <ConstraintProvider_ extends ConstraintProvider, Solution_> ConstraintVerifier<ConstraintProvider_, Solution_> build(
+            SolverConfig solverConfig) {
+        requireNonNull(solverConfig);
+        SolutionDescriptor<Solution_> solutionDescriptor = SolutionDescriptor
+                .buildSolutionDescriptor(requireNonNull((Class<Solution_>) solverConfig.getSolutionClass()),
+                        solverConfig.getEntityClassList().toArray(new Class<?>[] {}));
+        ScoreDirectorFactoryConfig scoreDirectorFactoryConfig = solverConfig.getScoreDirectorFactoryConfig();
+        ConstraintProvider_ constraintProvider = ConfigUtils.newInstance(null,
+                "constraintProviderClass",
+                (Class<ConstraintProvider_>) scoreDirectorFactoryConfig.getConstraintProviderClass());
```

Comment:
This formatting is weird. Pretty sure `constraintProviderClass` would have fit on the line above. Did you make the newline manually to force it to look like that?

Reply:
Fixed